### ISSUE TITLE
Where is backup file?

### ISSUE THREAD

**Issue by TheGP** [pgadmin-org/pgadmin4#8271]
After doing backup after it finishes the file is not present. Anyway how to access it if its on remote machine and I have no access to it with SSH.

**Comment by yogeshmahajan-1903** [comment]
If you are using pgadmin in server mode, then you can download the file from Tools > Storage Manager > Select file and click on download button. Please refer [this](https://www.pgadmin.org/docs/pgadmin4/8.14/storage_manager.html) link.

**Comment by TheGP** [comment]
![pgadmin4-tools](https://github.com/user-attachments/assets/039e23a9-2794-4337-8c3d-631bc516799d)

**Comment by TheGP** [comment]
No, just normal mode, desktop app which connects to the server.

**Comment by TheGP** [comment]
I dont see it there. grep doesnt find any files with such name either on local machine nor at server where I have access

**Comment by yogeshmahajan-1903** [comment]
@TheGP 
Can you please reopen backup dialog and click of file sign in filename field? pgadmin saves last directory used. So if you have not open file manager after taking backup, it should open the location which was used to store backup file.

<img width="894" alt="Screenshot 2024-12-18 at 10 17 35 AM" src="https://github.com/user-attachments/assets/906e7d40-88a5-4f3c-aee1-c18722f62a34" />

**Comment by TheGP** [comment]
`/app/bin/pg_dump --file "/home/eugenebos/test.sql" --host "1.1.1.1" --port "5433" --username "test" --no-password --format=c --large-objects --verbose --table "public.test" "test"`
So where is it?

```
cat /home/eugenebos/tokens.sql                                    (base) 
cat: /home/eugenebos/tokens.sql: No such file or directory
```

**Comment by yogeshmahajan-1903** [comment]
@TheGP 
File name specified in the command is test.sql however you are checking of tokens.sql

**Comment by TheGP** [comment]
```
cat test.sql                          (base) 
cat: test.sql: No such file or directory

```

**Comment by yogeshmahajan-1903** [comment]
As command indicate file is located at home directory it should be present there only. (I your case /home/eugenebos/test.sql). Are you searching with correct user?
Here is screenshot for my pgadmin desktop app on Mac OS.

<img width="954" alt="Screenshot 2024-12-19 at 10 12 10 PM" src="https://github.com/user-attachments/assets/781de734-598f-4c6e-9d39-1a147e1c8d9f" />

**Comment by TheGP** [comment]
I told u I tried "find" I do it from a user who has full access. There is no other users.

**Comment by yogeshmahajan-1903** [comment]
@TheGP 
We can connect over the call to figure out the issue between 9AM - 5 PM IST

**Comment by yogeshmahajan-1903** [comment]
No response from author, hence closing.

**Comment by TheGP** [comment]
Why closing? did u verify it works on ubuntu? issue still exists

**Comment by yogeshmahajan-1903** [comment]
Here is screenshot from ubuntu - 
<img width="1461" alt="Screenshot 2025-01-09 at 11 41 24 AM" src="https://github.com/user-attachments/assets/dfe2d6af-94b3-42cd-891a-92ec4c4f18a6" />

**Comment by TheGP** [comment]
That's so weird, ok, doesn't matter then, for me it doesn't work nor windows nor linux

**Comment by yogeshmahajan-1903** [comment]
@TheGP 
As I request earlier, we can connect one the call between 9AM - 5 PM IST.
